### PR TITLE
first commit .#4

### PR DESCRIPTION
**Pull request by MUAZAlimi**
@DevBigEazi am done with testimonial

### CODE REVIEW

File: testimonial.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+const handleTestimonial = (e) => {
+    e.preventDefault();
+    const data = new Date
```
Comment (DevBigEazi) [inline testimonial.js:3]: change variable name, data to date, and new Date syntax should be written correctly

**Review by DevBigEazi — CHANGES_REQUESTED** [review]
Pls make changes asap

**Comment by MUAZAlimi** [comment]
fixed

File: testimonial.js (hunk 1 above)
Comment (MUAZAlimi) [reply testimonial.js]: fixed

**Review by DevBigEazi — APPROVED** [review]
nice

**Pull request merged**